src/providers/oauth-provider.js

```diff
@@ -151,7 +151,6 @@ function OAuthProvider() {
 
       getRefreshToken() {
         var data = {
-          client_id: config.clientId,
```

Comment:
`id_client` is not required when request has authorization header. See more at <http://tools.ietf.org/html/rfc6749#section-6>

But, I will rollback this change, because it is not about "Revoke Token"